src/epsilon/main.d

```diff
@@ -38,6 +38,7 @@ void main(string[] args)
                     "slag", "Generate SLAG evaluator.", &slag,
                     "sweep", "Generate single-sweep evaluator.", &sweep,
                     "soag", "Generate SOAG evaluator.", &soag,
+                    "offset", "Additionally reports offset position, omits erroneous code output", &offset,
```

Comment:
What does "omits erroneous code output" mean?
Is `^` the problem? Or multi-line output in general?

What happens, if you use `--offset` together with `--verbose`?
Is the `trace` output ignored, or is it also "erroneous"?

Reply:
I only want to say that the erroneous code and the "^" pointer is omitted.
Do you have a better phrase?
... But may be this is not needed if we move to the option "--lsp". :smile: